tests/test_tests.py

```diff
@@ -0,0 +1,125 @@
+# placeholder file to test the testing capability
+
+import pytest
+
```

Comment:
Can we have a more descriptive name for the test file? The old name better described the purpose of the file, testing the cas 22.1.1 cost functions `test_cas220101_cost_functions.py`

Reply:
Done